navmazing/__init__.py

```diff
@@ -147,22 +168,15 @@ class NavigateToObject(object):
     callable that will navigate to the prerequisite step on the other object.
     """
 
-    def __init__(self, other_obj, target, obj=None):
```

Comment:
This is API changing - I'm sure this was in here for a reason - that we used it somewhere in CFME

Reply:
i recall this is just problematic code from the class based navigation, i beleive back then it tested well, as it was mainly used for passover - a crosschec should be done however

Reply:
i believe i vaguely recall finding a horrendous bug wrt naming too many things obj